dali/base/dautils.cpp

```diff
@@ -341,10 +341,15 @@ inline void normalizeScope(const char *name, const char *scope, unsigned len, St
     }
 }
 
-void normalizeNodename(const char *node, unsigned len, SocketEndpoint &ep, bool strict)
+void normalizeNodeName(const char *node, unsigned len, SocketEndpoint &ep, bool strict)
 {
     if (!strict)
-        skipSp(node);
+        while (isspace(*node))
```

Comment:
style/trivial: normally if it's a multi line block following a branch statement, we'd enclose in braces, i.e.:
```
    if (!strict)
    {
        while (isspace(*node))
        {
            node++;
            len--;
        }
    }
```